<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPage</class>
 <widget class="QWidget" name="SearchPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KLineEdit" name="queryEdit"/>
     </item>
     <item>
      <widget class="KComboBox" name="searchTypeBox">
       <property name="toolTip">
        <string>Search the collection by...</string>
       </property>
       <item>
        <property name="text">
         <string comment="The name of a book">Title</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="The writer of a book">Author</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="The books category">Genre</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="resultTable"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>resultTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>queryEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>725</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
